ocr_service/README.md

```
@@ -0,0 +1,100 @@
+```
+
+Request body:
+```json
+{
+  "image_url": "https://example.com/image.jpg"
+}
+```
+
+### Extract Text from PDF URL
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)